src/views/accounts/AccountTemplate/Detail/Account.vue

```diff
@@ -76,21 +76,28 @@ export default {
           name: {
             formatter: DetailFormatter,
             formatterArgs: {
-              route: 'AssetAccountDetail',
-              can: vm.$hasPerm('accounts.view_account')
+              drawer: true,
+              can: vm.$hasPerm('accounts.view_account'),
+              getRoute: ({ row }) => {
+                return {
+                  name: 'AssetAccountDetail',
+                  params: { id: row.id }
+                }
+              }
             }
           },
           asset: {
             label: this.$t('Asset'),
-            formatter: function(row) {
-              const to = {
-                name: 'AssetDetail',
-                params: { id: row.asset.id }
-              }
-              if (vm.$hasPerm('assets.view_asset')) {
-                return <router-link to={to}>{row.asset.name}</router-link>
-              } else {
-                return <span>{row.asset.name}</span>
+            formatter: DetailFormatter,
+            formatterArgs: {
+              drawer: true,
+              can: vm.$hasPerm('assets.view_asset'),
+              getTitle: ({ row }) => row.asset.name,
+              getRoute: ({ row }) => {
+                return {
+                  name: 'AssetDetail',
+                  params: { id: row.asset.id }
+                }
               }
             }
           },
```

Comment:
I'm sorry, it seems you have shared an incomplete code snippet. Could you please provide me with the original and modified versions of the codes? I'll be happy to review these side-by-side with you.